Dockerfile

```
@@ -0,0 +1,15 @@
+FROM rust:1.62 AS builder
+WORKDIR /tmp/
+COPY Cargo.toml Cargo.lock ./
+COPY indexer indexer
+COPY database database
+COPY circulating-supply circulating-supply
+RUN cargo build --release
+
+FROM ubuntu:20.04
+RUN apt update && apt install -yy openssl ca-certificates
+RUN apt-get install libpq5 -y
+COPY --from=builder /tmp/target/release/circulating-supply .
+COPY --from=builder /tmp/target/release/indexer-explorer-lake .
+COPY --from=builder /tmp/target/release/libexplorer_database.rlib .
+CMD [ "./indexer-explorer-lake" ]
```

Currently `chain_id` and `start_options` (`from-block`, `from-latest`, etc.) are supplied via command line arguments like so:
```sh
./indexer-explorer-lake testnet from-block 0
```
Because of the flexibility of these arguments, it isn't that straight forward to replace them with environment variables. I'm going to working on refactoring this to make supporting env vars simpler, but in the mean time I think we can just supply them like so:
```suggestion
ENTRYPOINT ./indexer-explorer-lake $CHAIN_ID $START_BLOCK_OPTIONS
```

This leans in to the idea of having two separate `Dockerfile`s as then we can explicitly specify different entrypoints for each.

What do you think?

---

Thanks. Changed to two Dockerfiles. PTAL.